docs/src/operate/c8y/supported-operations.md

```diff
@@ -140,40 +140,35 @@ The `Supported Operations API` can also be used to add support for custom operat
 For e.g: Cumulocity supports the `c8y_Command` that enables execution of shell commands on the device from the cloud.
 The same operation files can be used to define how the mapper should handle these operations when triggered from the cloud.
 
-### Adding new custom operation
+### Creating new custom operation
+
+Custom operations can be delivered by SmartREST message or JSON over MQTT depending on the content of the operation file. The former is limited to SmartREST input. The latter moves the actual execution of the operation to `tedge-agent` where is executed by the workflow.
+
+:::note
+The operation file needs to be readable by %%te%% user - `tedge` - and should have permissions `644`.
+The filename **MUST** follow the Cumulocity naming constraints on operation names: i.e. only use alphanumeric and underscore characters, e.g. `A-Z`, `a-z`, `0-9` and `_`. Alternatively, it can have `.template` extension for a master custom operation definition file.
```

Comment:
What is a *a master custom operation definition*?

Reply:
I admit this name can be confusing and says basically nothing so I changed it to _operation template file_.